compiler/rustc_metadata/src/native_libs.rs

```diff
@@ -387,6 +387,21 @@ impl<'tcx> Collector<'tcx> {
             let name = name.map(|(name, _)| name);
             let kind = kind.unwrap_or(NativeLibKind::Unspecified);
             let filename = find_bundled_library(name, verbatim, kind, cfg.is_some(), sess);
+
+            if matches!(
+                kind,
+                NativeLibKind::Static { bundle: Some(true), whole_archive: Some(true) }
+            ) && !features.packed_bundled_libs
+            {
+                feature_err(
+                    &sess.parse_sess,
+                    sym::packed_bundled_libs,
+                    m.span,
+                    "modifiers +bundle,+whole-archive are unstable",
+                )
+                .emit();
+            }
```

Comment:
Could you reuse the old error `IncompatibleLinkingModifiers` and its wording?
Also the error should be reported in `link.rs` in the same place as the old one, it just needs to be a feature error instead of a regular error now.